terraform-post-packer/main.tf

```diff
@@ -24,12 +24,12 @@ locals {
   account_name_regex  = format("^env[[:digit:]]+ \\(%s\\)$", local.images_account_type)
 }
 
-# The IDs of all cisagov/windows-commando-packer AMIs
+# The IDs of all cisagov/windows-commando-packer AMIs test
```

Comment:
Remove extra word:
```suggestion
# The IDs of all cisagov/windows-commando-packer AMIs
```

Reply:
@dav3r i know, i added that to test pre-commit. i forgot to remove it before pushing without pre-commit. it still fails with the same error locally for me when i reinstalled it. i removed it by uninstalling pre-commit again with this commit: [4db0c5e](https://github.com/cisagov/windows-commando-packer/pull/10/commits/4db0c5e8652984cf414760651cffdbd81888a8e8)

Reply:
I'm not quite following you here, but to clarify, when I said to commit without using pre-commit, I meant to commit with the [`--no-verify` flag](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---no-verify) in order to skip the pre-commit hooks locally, not to uninstall pre-commit.

Reply:
Oh nice. I wasn't aware of that flag. i was trying to achieve the same thing by uninstalling / reinstalling pre-commit. my local pre-commits pass now that there's an ami image created. I think i'm in a good place now

Reply:
@dav3r resolved with this commit [f354caf](https://github.com/cisagov/windows-commando-packer/pull/10/commits/f354cafd75d021c7ba76a427846b2a398cd9f7c1)

Reply:
Resolved in https://github.com/cisagov/windows-commando-packer/pull/10/commits/4db0c5e8652984cf414760651cffdbd81888a8e8.